src/components/Claim/Link/Initial.view.tsx

```
@@ -81,8 +83,9 @@ export const InitialClaimLinkView = ({
     const [routes, setRoutes] = useState<any[]>([])
     const [inputChanging, setInputChanging] = useState<boolean>(false)
     const [showConfirmationModal, setShowConfirmationModal] = useState<boolean>(false)
+    const [showConfirmationModal, setShowConfirmationModal] = useState<boolean>(false)
```

Biome (1.9.4)

[error] Shouldn't redeclare 'showConfirmationModal'. Consider to delete it or rename it.

'showConfirmationModal' is defined here:


(lint/suspicious/noRedeclare)